schemas/ships.json

```diff
@@ -1156,6 +1026,16 @@
                 "examples": [
                   1
                 ]
+              },
```

Comment:
Would this also be used to indicate you cannot build a mf/d3/legio until you have shipyard 26?

Reply:
Tier locked by ops is used for ships that are bound to ops level. For example, the USS Franklin can't be maxed until a certain level because you only get those warp cores when your ops level is higher. Same goes for the Sarcophagus. 

If they need a shipyard 26, that will be added to `requirements`.

Reply:
some ships have more then one teir lock. is that accounted for?

Reply:
The tier locked by ops is saved in the tier schema. So basically every tier could be locked when needed.